query-engine/core/src/telemetry/helpers.rs

```diff
@@ -0,0 +1,112 @@
+use super::models::TraceSpan;
+use once_cell::sync::Lazy;
+use opentelemetry::sdk::export::trace::SpanData;
+use opentelemetry::trace::{TraceContextExt, TraceId};
+use opentelemetry::Context;
+use serde_json::{json, Value};
+use std::collections::HashMap;
+use tracing::{Metadata, Span};
+use tracing_opentelemetry::OpenTelemetrySpanExt;
+use tracing_subscriber::EnvFilter;
+
+pub static SHOW_ALL_TRACES: Lazy<bool> = Lazy::new(|| match std::env::var("PRISMA_SHOW_ALL_TRACES") {
+    Ok(enabled) => enabled.to_lowercase() == *("true"),
+    Err(_) => false,
+});
+
+pub fn spans_to_json(spans: Vec<SpanData>) -> String {
+    let json_spans: Vec<Value> = spans.into_iter().map(|span| json!(TraceSpan::from(span))).collect();
+    let span_result = json!({
+        "span": true,
+        "spans": json_spans
+    });
+
+    match serde_json::to_string(&span_result) {
+        Ok(json_string) => json_string,
+        Err(_) => "".to_string(),
+    }
```

Comment:
Not a substantial improvement, but I think the whole `match` can be replaced with `serde_json::to_string(&span_result).unwrap_or_default()`.